.github/workflows/unit_tests_with_released_puppet_gem.yaml

```diff
@@ -34,13 +34,7 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.ruby }}
-
-      - name: Prepare testing environment with bundler
-        run: |
-          git config --global core.longpaths true
-          bundle config set system 'true'
-          bundle config set --local without 'release'
```

Comment:
Does it matter that we lost these configurations? It's possible to set these as environment variables (at least for the latter two, not sure about global git config)?

Reply:
I don't think we need the longpaths configuration anymore--it was initially implemented because of the GitHub Changelog Generator repo: https://github.com/puppetlabs/puppetlabs-augeas_core/pull/35

But we no longer use that Changelog Generator anymore, so I don't think we need the option.

This action is used by the core modules and the puppet_agent module, none of which have a "release" group in their Gemfiles. We won't lose anything by removing this configuration.

The Bundler docs don't seem to have anything on `bundle config set system 'true'`, but it seems like it sets all future Bundler configuration to take place at the system level (as opposed to local/global). If that's the case, then we're not doing any additional Bundler configuration so it doesn't matter anyway.